fxa-auth-db-server/docs/DB_API.md

```diff
@@ -249,7 +250,7 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt
+* sessionToken : data, uid, createdAt, userAgent, os, deviceType
```

Comment:
Okay, so we have:

* `uaBrowser`, e.g. `Firefox`.
* `uaBrowserVersion`, e.g. `42`.
* `uaOS`, e.g. `Android`.
* `uaOSVersion`, e.g. `5.1`.
* `uaDeviceType`, e.g. `mobile`.

Look good?

One more question: Where `ua-parser` returns `Other`, we want to replace that with the empty string right? Makes it easier to eyeball the data and matches up with the default value in SQL.